ui/js/values.js

```diff
@@ -11,6 +11,7 @@ treeherder.value("thPlatformMap", {
     "linux64-qr": "Linux x64 QuantumRender",
     "linux64-nightly": "Linux x64 Nightly",
     "linux64-stylo": "Linux x64 Stylo",
+    "linux64-stylo-sequential": "Linux x64 Stylo-Seq",
```

Comment:
Putting this as "Linux x64 Stylo Sequential" makes the debug group break onto two lines, so I cut this to just "Seq" to prevent that. Happy to go either way, though.